### DISCUSSION TITLE
Remove OPTIONS calls from log

### DISCUSSION THREAD

**Discussion by DavidP-B** [moleculerjs/moleculer#1076] [Q&A] [answered]
Hi, can someone tell me if it possible to remove from log info OPTIONS calls and maintain only GET, POST, ... calls?

Thx!

**Comment by icebob** [comment, marked as answer]
You can overwrite the [`logRequest(req)`](https://github.com/moleculerjs/moleculer-web/blob/master/src/index.js#L961) and [`logResponse(req, res, data)`](https://github.com/moleculerjs/moleculer-web/blob/master/src/index.js#L995) method in API gw service.

**Comment by DavidP-B** [comment]
Perfect, works like a charm!
